packages/cli/src/sso.ee/saml/saml-validator.ts

```diff
@@ -22,7 +25,20 @@ export class SamlValidator {
 	}
 
 	async validateMetadata(metadata: string): Promise<boolean> {
-		return await this.validateXml('metadata', metadata);
+		const validXML = await this.validateXml('metadata', metadata);
+
+		if (validXML) {
+			const idp = IdentityProvider({
+				metadata,
+			});
+
+			const binding = idp.entityMeta.getSingleSignOnService('redirect');
```

Comment:
I see samlify exports a Constants object, you could maybe use it here: Constants.wording.binding.redirect